.github/workflows/pull-request-comments.yml

```diff
@@ -163,3 +163,46 @@ jobs:
             `;
 
             github.rest.issues.createComment( commentInfo );
+
+  # Leaves a comment on a pull request when no Trac ticket is included in the pull request description.
+  trac-ticket-check:
+    name: Comment on a pull request when no Trac ticket is included
+    runs-on: ubuntu-latest
+    permissions:
+      issues: write
+      pull-requests: write
+    if: ${{ github.repository == 'WordPress/wordpress-develop' && github.event_name == 'pull_request_target' }}
+    steps:
+      - name: Check for Trac ticket and comment if missing
+        uses: actions/github-script@60a0d83039c74a4aee543508d2ffcb1c3799cdea # v7.0.1
+        with:
+          script: |
+            const pr = await github.rest.pulls.get( {
+              owner: context.repo.owner,
+              repo: context.repo.repo,
+              pull_number: context.issue.number,
+            } );
+
+            const prBody = pr.data.body ?? '';
+
+            const tracTicketRegex = new RegExp( 'https?://core.trac.wordpress.org/ticket/([0-9]+)', 'g' );
+            const tracTicketMatches = prBody.match( tracTicketRegex );
```

Comment:
It's not obvious (or recommended), but it is possible to also include the Trac ticket URL in the PR title. I tested this to confirm in #6088. You'll notice it's listed on the Trac ticket. We should check for it just in case, but let's not recommend this practice.